project/Dependencies.scala

```diff
@@ -55,11 +56,14 @@ object Dependencies {
   val ScalaPool              = "io.github.andrebeat"        %% "scala-pool"                % Versions.ScalaPool
   val ScalaTestCore          = "org.scalatest"              %% "scalatest-core"            % Versions.ScalaTest
   val SimpleJmx              = "com.j256.simplejmx"          % "simplejmx"                 % Versions.SimpleJmx
-  val Specs2Common           = "org.specs2"                 %% "specs2-common"             % Versions.Specs2
-  val Specs2Core             = "org.specs2"                 %% "specs2-core"               % Versions.Specs2
-  val Specs2JUnit            = "org.specs2"                 %% "specs2-junit"              % Versions.Specs2
-  val Specs2Matcher          = "org.specs2"                 %% "specs2-matcher"            % Versions.Specs2
-  val Specs2ScalaCheck       = "org.specs2"                 %% "specs2-scalacheck"         % Versions.Specs2
+  val Specs2Common_4         = "org.specs2"                 %% "specs2-common"             % Versions.Specs2_4
```

Comment:
Nitpick: If we name this `Specs2_4Common` and `Specs2_5Common` (and the rest of the libraries) we get each specs2 version grouped when lexicographically sorting these lines.